kyverno-test-util.py

```diff
@@ -0,0 +1,132 @@
+import sys
+import yaml
+import os
+from types import NoneType
+
+# ReplicaSet is not here, because I have no idea how to change policies to make it work
+allowedResources = ['Job', 'Deployment', 'StatefulSet']
+ignoredFileNames = ['prometheus-scrapeConfig', 'chart', 'Chart',
+                    'values', 'CustomResourceDefinition', 'crds']
+
+kyvernoTestFileName = 'kyverno-test.yaml'
+
+def deleteLeftovers():
+    if os.path.exists(kyvernoTestFileName):
+        os.remove(kyvernoTestFileName)
+
+def writeTitle():
+    title = {'name': 'Kyverno test'}
+    appendToFile(title)
+
+def writePolicies(policies_location):
+    policies = {'policies': f'{policies_location}'}
```

Comment:
```suggestion
    policies = {'policies': [f'{policies_location}'}]
```
I am using kyverno cli Version 1.9.2.
It complains if this is not a list.